packages/_site/src/pages/components/loading/index.tsx

```diff
@@ -98,6 +98,56 @@ export default function LoadingExample() {
         <LoadingSpinner size={LoadingSize.SMALL} />
 	);
 }
+`}
+                                />
+                            </div>
+                        </div>
+                        <div>
+                            <ExampleBlock
+                                title="Default"
+                                description="The medium version of the loading component."
+                            />
+                            <div {...stylex.props(styles.codeWrapper)}>
+                                <PreviewBlock>
+                                    <LoadingSpinner size={LoadingSize.MEDIUM} />
+                                </PreviewBlock>
+
+                                <CodeBlock
+                                    language="tsx"
+                                    code={`import React from 'react';
+
+import { LoadingSpinner, LoadingSize } from "@controlkit/loading";
+
+export default function LoadingExample() {
+	return (
+        <LoadingSpinner size={LoadingSize.MEDIUM} />
+	);
+}
+`}
+                                />
+                            </div>
+                        </div>
+                        <div>
+                            <ExampleBlock
+                                title="Default"
```

Comment:
If you're adding properties to the component than it isn't default, name it after the variant form.